source/MaterialXView/Main.cpp

```diff
@@ -234,7 +258,10 @@ int main(int argc, char* const argv[])
                                             screenWidth,
                                             screenHeight,
                                             screenColor,
-                                            multiSampleCount);
+                                            multiSampleCount,
+                                            bakeWidth,
```

Comment:
Rather than extending the Viewer constructor, I would recommend making these Viewer accessors, as we've done with modifiers such as setMeshRotation and setCameraPosition.